contributing/release-schedule.md

```diff
@@ -0,0 +1,26 @@
+# Release Schedule
+
+We push a new release of AMP to all AMP pages every week on Thursday. The more detailed schedule is as follows:
+
+- Every Wednesday we cut a green release from our `master` branch.
```

Comment:
Maybe instead of green say "we cut the release at the latest commit that passed all tests."